commit e8f591165f8ff077a183ff02147beb4129519f7d
Author: Thomas Woerner <twoerner@redhat.com>
Date:   Wed Sep 28 13:08:49 2016 +0200

    Drop debug output for tmpint in getopt_long while loop

diff --git a/iptstate.cc b/iptstate.cc
index b03e566..5c9ddb1 100644
--- a/iptstate.cc
+++ b/iptstate.cc
@@ -2179,7 +2179,6 @@ int main(int argc, char *argv[])
   // Command Line Arguments
   while ((tmpint = getopt_long(argc, argv, "Cd:D:hlmcoLfpR:r1b:s:S:tv",
                                long_options, &option_index)) != EOF) {
-    printf("loop: %d\n", tmpint);
     switch (tmpint) {
     case 0:
       /* Apparently this test is needed?! Seems lame! */
